static/js/compose.js

```diff
@@ -423,6 +429,8 @@ export function initialize() {
         } else {
             $("#compose_close").attr("data-tooltip-template-id", "compose_close_tooltip");
         }
+
+        compose_validate.check_send_permissions();
```

Comment:
> Node tests seems fine now.
> 
> I locally tested the PR. The button is disabled as expected when I click open the compose box for a stream to which I am not allowed to post,but when I start typing something in the compose box the send button is enabled again. And same happens for PMs

After a little sleuthing, I found that this function was changed and merged recently into main (commit hash: `b8a760b14e5`). When I rebased my commits on main, `compose_validate.check_send_permissions()` ended up before `compose_validate.check_overflow_text()`. Since both these functions can enable/disable the send button the logical order is important and I failed to notice the ordering changed. This is now corrected in the latest and works as expected